vagrant/requirements.txt

```diff
@@ -20,3 +20,5 @@ pydot>=1.1.0,<1.2
 sqlalchemy-schemadisplay>=1.3,<1.4
 psycopg2>=2.6.1,<2.7
 wrapt>=1.10.8,<1.11
+# only needed for ldap caching
+ldap3==2.1.0
```

Comment:
Is there a better way to do this? Since `ldap3` is only needed for caching the ldap, including it in the global requirements seems inappropriate (but works, obviously).